quotefault/__init__.py

```diff
@@ -1,23 +1,38 @@
-# import all relevant packages
+"""
+File name: __init__.py
+Author: Nicholas Mercadante
+Contributors: Devin Matte, Max Meinhold, Joe Abbate
+"""
 import os
 import subprocess
 from datetime import datetime
 
 import requests
 from csh_ldap import CSHLDAP
 from flask import Flask, render_template, request, flash, session, make_response
+from flask_migrate import Migrate
 from flask_pyoidc.flask_pyoidc import OIDCAuthentication
 from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy import func
+from sqlalchemy.orm.query import Query
+from werkzeug.utils import redirect
+
+from config import LDAP_BIND_DN

```

Comment:
Why is this imported instead of using app.config syntax?

Reply:
This was fixed in one of my later commits